.github/workflows/e2e.yml

```diff
@@ -20,32 +20,38 @@ jobs:
 
       - name: Cache build
         uses: actions/cache@v2.1.6
+        id: build-cache
         with:
           path: build
-          key: build-${{ hashFiles('src/**') }}
+          key: build-${{ github.sha }}
```

Comment:
What's the advantage of using the commit sha over the source file hash?

Reply:
If we use only "src" then the CI would treat commits with the same source but e.g. different package versions as cache hit. Therefore we would have to include every file hash that could influence the build outcome. So if we miss to adjust this key because we e.g. add a new file or env variable then we risk corrupting the e2e test job.
This risk is too high for me. Therefore we should just use the commit hash.
The main purpose of this PR is to speed up tests that get restarted without change.

Reply:
I see, thanks for the explaination.